### PR TITLE
Create GraphQL Topic for Wish List Endpoint

### LINKED ISSUE #3341: GraphQL: Add wishlist endpoint for 2.3.1
# Community engineering issue

Type of Doc:
- [x] New topic
- [ ] Topic Update, Devdoc link:
- [ ] Other

Magento 2 GitHub Issue: https://github.com/magento/graphql-ce/issues/43

Magento 2 GitHub PR: https://github.com/magento/graphql-ce/pull/223

Magento 2 version:
- [ ] 2.1.X
- [ ] 2.2.X
- [x] 2.3.X
- [ ] back port
- [ ] forward port

Notifications of Changes:
- [ ] Changes to Admin interface?
- [ ] New software requirements?
- [x] New REST endpoints or updated GraphQL query syntax?
- [ ] New command line option?

## Content

The `WishlistGraphQl` module currently defines one query, which returns the contents of a customer's wishlist. There are no mutations at this time,

### PR DESCRIPTION
**Pull request by erikmarr**
## This PR is a:

- [x] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will fix #3341 .

## Additional information

List all affected URLs 

- New topic

### CODE REVIEW

File: guides/v2.3/graphql/reference/wishlist.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Query structure
+
+`wishlist: WishlistOutput`
+
+### Wish list attributes
+
+Attribute | Data type | Description
+--- | --- | ---
+`items` | [Wishlistitem] | An array of items in the customer's wish list
```
Comment (keharper) [inline guides/v2.3/graphql/reference/wishlist.md]: Link Wishlistitem to the H3 below

File: guides/v2.3/graphql/reference/wishlist.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+`sharing_code` | String | An encrypted code that Magento uses to link to the wish list
+`updated_at` | String | The time of the last modification to the wish list
+
+### Wish list item attributes
+Attribute | Data type | Description
+--- | --- | ---
+`added_at` | String | The time when the customer added the item to the wish list
+`description` | String | The customer's comment about this item
+`id` | Int | The wish list item ID
+`product` | ProductInterface | The ProductInterface contains attributes that are common to all types of products. Note that descriptions may not be available for custom and EAV attributes.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/wishlist.md]: Link ProductInterface to `guides/v2.3/graphql/reference/products.html#ProductInterface`

File: guides/v2.3/graphql/reference/wishlist.md (hunk 1 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/wishlist.md]: Added anchor link.

File: guides/v2.3/graphql/reference/wishlist.md (hunk 2 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/wishlist.md]: Added link to ProductInterface.

File: guides/v2.3/graphql/reference/wishlist.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+group: graphql
+title: Wish list endpoint
+---
+
+The Wish list endpoint returns the contents of a customer's wish list.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/wishlist.md]:
```suggestion
The Wish list endpoint defines the contents of a customer's wish list.
```

Comment (keharper) [reply guides/v2.3/graphql/reference/wishlist.md]: Create a "Query" H2 and a bridge sentence describing the query.

File: guides/v2.3/graphql/reference/wishlist.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+          "product": {
+            "sku": "MSH11",
+            "name": "Arcadio Gym Short"
+          }
+        }
+      ]
+    }
+  }
+}
+```
```
Comment (keharper) [inline guides/v2.3/graphql/reference/wishlist.md]: Create a "Mutatations" H2 and say they'll be added in a future release.

Comment (erikmarr) [reply guides/v2.3/graphql/reference/wishlist.md]: done

File: guides/v2.3/graphql/reference/wishlist.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    "wishlist": {
+      "items_count": 2,
+      "name": "Wish List",
+      "sharing_code": "KAXDj0HlM7Y2s58mllsVhSJvRj4fWIZj",
+      "updated_at": "2019-02-13 22:47:45",
+      "items": [
+        {
+          "id": 1,
+          "qty": 1,
+          "description": null,
```
[the remark this answers is not in the document]
Comment (erikmarr) [reply guides/v2.3/graphql/reference/wishlist.md]: fixed

File: guides/v2.3/graphql/reference/wishlist.md (hunk 3 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/wishlist.md]: done

File: _data/toc/graphql.yml [hunk 6]
```diff
@@ -79,3 +79,6 @@ pages:
 
             - label: urlResolver endpoint
               url: /graphql/reference/url-resolver.html
+
+            - label: Wish list endpoint
```
Comment (keharper) [inline _data/toc/graphql.yml]:
```suggestion
            - label: Wishlist endpoint
```

File: guides/v2.3/graphql/reference/wishlist.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
+---
+group: graphql
+title: Wish list endpoint
```
Comment (keharper) [inline guides/v2.3/graphql/reference/wishlist.md]:
```suggestion
title: Wishlist endpoint
```

File: guides/v2.3/graphql/reference/wishlist.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+---
+
+The Wish list endpoint defines the contents of a customer's wish list.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/wishlist.md]:
```suggestion
The Wishlist endpoint defines the contents of a customer's wish list.
```

File: guides/v2.3/graphql/reference/wishlist.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+## Query
+Use Wish list queries to retrieve information about a customer's wish list.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/wishlist.md]:
```suggestion
Use `wishlist` queries to retrieve information about a customer's wish list. 
```

Comment (erikmarr) [reply guides/v2.3/graphql/reference/wishlist.md]: fixed

File: guides/v2.3/graphql/reference/wishlist.md (hunk 8 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/wishlist.md]: fixed

File: guides/v2.3/graphql/reference/wishlist.md (hunk 7 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/wishlist.md]: fixed

File: _data/toc/graphql.yml (hunk 6 above)
Comment (erikmarr) [reply _data/toc/graphql.yml]: fixed

**Comment by erikmarr** [comment]
running tests

**Pull request merged**